tests/test_wave_01.py

```diff
@@ -161,15 +153,12 @@ def test_delete_task_not_found(client):
     # Assert
     assert response.status_code == 404
 
-    raise Exception("Complete test with assertion about response body")
-    # *****************************************************************
-    # **Complete test with assertion about response body***************
-    # *****************************************************************
+    assert response_body == {'message': 'Task 1 not found'}
 
-    assert Task.query.all() == []
+    assert Task.query.all() == []# why is this?
```

Comment:
When tests are being run, we have a fixture that puts one record into the test DB. After you delete that record, if you query for it again then it will return an empty list. 

Your assertion tests that the response to the client is formatted correctly, but we want to ensure that something was actually deleted from the database and the way we do that is by checking the database